test/xchainGauges/childGauge.ts

```diff
@@ -347,6 +354,24 @@ describe("ChildGauge", () => {
           "14285879629629599900",
         )
       })
+      it("Successfully claims rewards with an additional 0 rate reward token", async () => {
+        await childGauge.add_reward(dummyRewardToken2.address, users[1])
+
+        await childGauge
+          .connect(signers[1])
+          ["deposit(uint256)"](BIG_NUMBER_1E18.mul(100))
+        await increaseTimestamp(DAY)
+
+        // 100 tokens distributed over 1 week, 1 day passed
+        // 100 * 1e18 / 7
+        await expect(
+          childGauge.connect(signers[1])["claim_rewards()"](),
+        ).to.changeTokenBalance(
+          dummyRewardToken,
+          users[1],
+          "14285879629629599900",
+        )
+      })
```

Comment:
Hmm I believe the setup for the edge case we want to test against is:
`add_reward(randomEOA, rewardForwarder)`

Case where we use an EOA address instead of an ERC20 address for the token. The main reason for this test is to see if the ChildGauge contract would make any calls to `randomEOA` that would fail due to the fact that it is NOT erc20 or a contract at all.

Although after manually looking into ChildGauge, I think it wouldn't make any call to `randomEOA` since `total_claimable` will always be 0, we should try to confirm this via test.

https://github.com/saddle-finance/saddle-contract/blob/e7b2258152df959ce26c85e03a6cd3f34f2a0577/contracts/xchainGauges/implementations/ChildGauge.vy#L226-L252